<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>832</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>832</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>832</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VentanaPrincipal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#VentanaPrincipal{
	border-image: url(:fondo.jpg);	
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Maxima</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OfLlenadora1">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>440</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OfLlenadora1
{
	background-color: transparent;
	border-image: url(:off.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Maxima</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maxima</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OfPersona1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OfPersona1{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="velocidadLimpiadora">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OfLimpiadora">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OfLimpiadora
{
	background-color: transparent;
	border-image: url(:off.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OfPersona2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>720</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OfPersona2
{
	background-color: transparent;
	border-image: url(:off.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>580</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OnLimpiadora">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OnLimpiadora
{
	background-color: transparent;
	border-image: url(:on.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Max3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Max4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>550</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OnPersona2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>661</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OnPersona2
{
	background-color: transparent;
	border-image: url(:on.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Actu2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Onpersona1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_Onpersona1
{
	background-color: transparent;
	border-image: url(:on.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pause">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Max1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OnLlenadora2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OnLlenadora2
{
	background-color: transparent;
	border-image: url(:on.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="velocidadPersona2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>560</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Maxima</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OfLlenadora2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OfLlenadora2
{
	background-color: transparent;
	border-image: url(:off.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="velocidadPersona1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OnLlenadora1">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>380</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OnLlenadora1
{
	background-color: transparent;
	border-image: url(:on.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_resume">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fábrica de botellas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Max2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Actu1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Actu3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>280</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Actu4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>580</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OfTapadora">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>720</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OfTapadora
{
	background-color: transparent;
	border-image: url(:off.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OnTapadora">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>660</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_OnTapadora
{
	background-color: transparent;
	border-image: url(:on.png);	
	background: none;
	border: none;
	background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="velocidadLlenadora2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>610</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="velocidadTapadora">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>630</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>330</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="velocidadLlenadora1">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>350</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_Tipos_Botellas">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>140</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Tipo botella</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Medida</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Probabilidad</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="line_tipo">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>80</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Tipo de botella</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_cantidad_botellas">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>80</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Medida</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ingresar_tabla">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>110</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_Rechazo_Botellas">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>640</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Tipo botella</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Aceptadas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Desechadas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>% Rechazo</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="line_probabilidad">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>80</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Probabilidad</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_Persona1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Tipo botella</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tabla_Limpiadora">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Tipo botella</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tabla_Llenadora2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Tipo botella</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tabla_Llenadora1">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>360</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Tipo botella</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tabla_Tapadora">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>640</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Tipo botella</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>730</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Maxima</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>750</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Max5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>720</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_Actu5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>750</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Banda transportadora 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Carga botellas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiadora</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Banda transportadora 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Banda transportadora 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>680</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Banda transportadora 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>520</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Banda transportadora 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>570</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tapadora</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Inspector de calidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>330</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Llenadora 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Llenadora 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
